routes/web.php

```diff
@@ -1,7 +1,11 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\CommentController;
 
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::resource('comments', CommentController::class);
```

Comment:
A resource is a wildcard here. Specify the methods you want to use specifically so that Laravel blocks the rest.

Reply:
Done